lib/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueue.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.hmpps.sqs
+
+import com.amazonaws.services.sqs.AmazonSQS
+
+class HmppsQueue(
+  val sqsAwsClient: AmazonSQS,
+  val queueName: String,
+  val sqsAwsDlqClient: AmazonSQS,
+  val dlqName: String,
+) {
+  val queueUrl: String by lazy { sqsAwsClient.getQueueUrl(queueName).queueUrl }
+  val dlqUrl: String by lazy { sqsAwsDlqClient.getQueueUrl(dlqName).queueUrl }
+}
```

Comment:
This assumes all queues have a dlq which is always true afaik.  If we see a use case for a queue without a dlq then we can make the dlq nullable.

Reply:
I think court-register uses a queue to send a message, not receive - so no DLQ. But maybe they don't need this library

Reply:
Cool.  I think court-register will need this library when we start implementing the queue health indicators so we can tackle it then.